entrypoint.sh

```diff
@@ -55,8 +55,8 @@ echo "INPUT_LOG_LEVEL: ${INPUT_LOG_LEVEL}"
 echo "----------------------------------"
 
 echo "Setting Git safe directory (CVE-2022-24765)"
-echo "git config --global --add safe.directory ${INPUT_SOURCE}"
-git config --global --add safe.directory "${INPUT_SOURCE}"
+echo "git config --global --add safe.directory ${GITHUB_WORKSPACE}"
```

Comment:
is this a GitHub environment variable? If so, does it need to follow the double bracket syntax? i.e. `${{ GITHUB_WORKSPACE }}`

Reply:
${{ ... }} works on GitHub Workflow yaml files only